ui/app/appointments/views/manage/list/defaultListPrint.html

```diff
@@ -80,7 +82,9 @@ <h5 ng-if="!searchedPatient">{{::'APPOINTMENT_DATE' | translate}} {{::startDate
                     <td class="table-mid-width" ng-if="enableServiceTypes">{{::appointment.serviceType && (appointment.serviceType.name + ' [' + appointment.serviceType.duration + ' min]')}}</td>
                     <td class="table-mid-width">{{::appointment.location.name}}</td>
                     <td>{{::appointment.status}}</td>
-                    <!--<td class="notes">{{::appointment.comments}}</td>-->
+                   <!-- <td ng-show="hasAdditionalInfoColumnsEntered()" class="table-mid-width">{{display(appointment.additionalInfo)}}</td>
+                    <td ng-repeat-start="columnValue in additionalInfoColumns" class="table-mid-width" ng-repeat-end>{{displayNewColumns(appointment.additionalInfo, columnValue)}}</td>
+                    <td class="notes">{{::appointment.comments}}</td>-->
```

Comment:
Remove commented lines

Reply:
These commented statements were purposely retained because, in case, any implementor wants to have those columns, he/she can just uncomment the same and use.